mautic/mautic
# Removing migrations created for Mautic 4

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

[The migrations created in Mautic 4](https://github.com/mautic/mautic/tree/4.4/app/migrations) are not necessary anymore. The old instances have run them and the new instances do not need them as the schema is generated from fresh entities.

We did the same thing form Mautic 4.0.0 and there were no issues AFAIK.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. I don't think this can be tested. Review is sufficient.
